src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py

```diff
@@ -588,139 +588,114 @@ def test_graph_user_scenario(self):
 
 class GraphGroupScenarioTest(ScenarioTest):
 
+    def clean_resource(self, resource, is_user=False):
+        try:
+            if is_user:
+                self.cmd('ad user delete --id {}'.format(resource))
+            else:
+                self.cmd('ad group delete -g {}'.format(resource))
+        except Exception:
+            pass
+
     def test_graph_group_scenario(self):
         username = get_signed_in_user(self)
         if not username:
             return  # this test delete users which are beyond a SP's capacity, so quit...
 
         domain = 'AzureSDKTeam.onmicrosoft.com'
         self.kwargs = {
-            'user1': self.create_random_name(prefix='deleteme1', length=15),
-            'user2': self.create_random_name(prefix='deleteme2', length=15),
-            'domain': domain,
-            'new_mail_nick_name': 'deleteme11',
-            'group': 'deleteme_g',
-            'pass': 'Test1234!!'
+            'group': self.create_random_name(prefix='testgroup', length=24),
+            'mail_nick_name': 'deleteme11',
+            'child_group': self.create_random_name(prefix='testchildgroup', length=24),
+            'leaf_group': self.create_random_name(prefix='testleafgroup', length=24),
+            'user1': self.create_random_name(prefix='testgroupuser1', length=24),
+            'user2': self.create_random_name(prefix='testgroupuser2', length=24),
+            'pass': 'Test1234!!',
+            'domain': domain
         }
         self.recording_processors.append(AADGraphUserReplacer('@' + domain, '@example.com'))
         try:
-            # create user1
-            user1_result = self.cmd('ad user create --display-name {user1} --password {pass} --user-principal-name {user1}@{domain}').get_output_in_json()
-            self.kwargs['user1_id'] = user1_result['id']
-
-            # update user1
-            self.cmd('ad user update --display-name {user1}_new --account-enabled false --id {user1}@{domain} --mail-nickname {new_mail_nick_name}')
-            user1_update_result = self.cmd('ad user show --upn-or-object-id {user1}@{domain}', checks=[self.check("displayName", '{user1}_new')]).get_output_in_json()
-            self.cmd('ad user update --id {user1}@{domain} --password {pass}')
-            self.cmd('ad user update --id {user1}@{domain} --password {pass} --force-change-password-next-login true')
-            with self.assertRaises(CLIError):
-                self.cmd('ad user update --id {user1}@{domain} --force-change-password-next-login false')
```

Comment:
@calvinhzy, please make sure these removed tests are covered by user tests.

Reply:
Ah. I saw you have already mentioned that: https://github.com/Azure/azure-cli/pull/21663#discussion_r827775708/ 😄